pymc3/variational/updates.py

```diff
@@ -117,10 +117,28 @@
     "adam",
     "adamax",
     "norm_constraint",
-    "total_norm_constraint"
+    "total_norm_constraint",
+    "Sgd",
+    "Momentum",
+    "NesterovMomentum",
+    "Adagrad",
+    "RMSProp",
+    "AdaDelta",
+    "AdaMax",
+    "Adam",
 ]
 
 
+class Optimizer(object):
```

Comment:
an extension is to add the `__doc__` of the function to the class to also get the docs.